src/main/java/com/powsybl/openloadflow/equations/Equation.java

```diff
@@ -123,10 +123,10 @@ public Equation addTerms(List<EquationTerm> terms) {
 
     private static double getBusTargetV(LfBus bus) {
         Objects.requireNonNull(bus);
-        if (bus.isDiscreteVoltageControlled()) {
-            return bus.getDiscreteVoltageControl().getTargetValue();
-        }
-        return getVoltageControlledTargetValue(bus).orElse(Double.NaN);
+        return bus.getDiscreteVoltageControl().filter(dvc -> bus.isDiscreteVoltageControlled())
+            .map(DiscreteVoltageControl::getTargetValue)
+            .orElse(getVoltageControlledTargetValue(bus)
+                .orElse(Double.NaN));
```

Comment:
Indentation issue here?

Reply:
No, but it's a bit unclear, so I changed it to 
```
            .orElse(
                getVoltageControlledTargetValue(bus).orElse(Double.NaN)
            );
```